WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -75,6 +80,18 @@ public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 						MinAnonScoreTarget = x - 1;
 					}
 				});
+
+		this.ValidateProperty(x => x.PlebStopThreshold, ValidatePlebStopThreshold);
+		this.WhenAnyValue(x => x.PlebStopThreshold)
+			.Subscribe(
+			x =>
+			{
+				if (Money.TryParse(x, out Money result))
+				{
+					wallet.KeyManager.PlebStopThreshold = result;
+					wallet.KeyManager.ToFile();
+				}
+			});
```

Comment:
```suggestion
this.WhenAnyValue(x => x.PlebStopThreshold)
			.Skip(1)
			.Throttle(TimeSpan.FromMilliseconds(1000))
			.ObserveOn(RxApp.TaskpoolScheduler)
			.Subscribe(x =>
			{
				if (Money.TryParse(x, out Money result) && result != wallet.KeyManager.PlebStopThreshold)
				{
					wallet.KeyManager.PlebStopThreshold = result;
					wallet.KeyManager.ToFile();
				}
			});
```

- Skip for preventing unnecessary check when ViewModel created
- Throttle to check only when the user stopped typing
- ObserveOn I think this is unnecessary because of the throttle, but just to make sure it doesn't happen on the UI thread.
- `result != wallet.KeyManager.PlebStopThreshold)` no need to save when the value is the same.